code-challenges.js

```diff
@@ -41,16 +71,24 @@ var vowelTester1 = "learn"
 var vowelTester2 = "throw"
 // Expected output: 3
 
+const firstVow =("") {
+var vowTestArr =  vowelTester1.split
+  for newArr (let i = 0; i < vowelTester1.length, i++){
+    char c = vowelTester1.charAt(i)
+    if (isVowel(c))
+      return vowelTester1.indexOf(c)
+  }
 
-
-
-
+}
+console.log(firstVow(vowelTester1))
 // --------------------5) Create a function that takes the toonimals array and returns an array with only the toon objects that are cats.
 // Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
 
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
+const toonArr = (toonimals){
 
+}
```

Comment:
Looks like we're missing most of the code for this function. Let me know if this was an upload issue!